config.yaml

```diff
@@ -33,47 +33,47 @@ timeline:
 # Using Home Assistant conventions to ease upcoming integration
 sources:
 
-  # direct support for raspberry picamera
-  picamera:
-    uri: picamera
-    type: video
-    live: true
-
-  # local video device integration example
-  webcam:
-    uri: /dev/video0
-    type: video
-    live: true
+#  # direct support for raspberry picamera
+#  picamera:
+#    uri: picamera
+#    type: video
+#    live: true
+#
+#  # local video device integration example
+#  webcam:
+#    uri: /dev/video0
+#    type: video
+#    live: true
 
   recorded_cam_feed:
     uri: file:///workspace/tests/pipeline/avsource/test2-cam-person1.mkv
-#    type: video
+    type: video
 #    live: true
 
 ai_models:
   image_detection:
     model:
-      tflite: /opt/ambianic-edge/ai_models/mobilenet_ssd_v2_coco_quant_postprocess.tflite
```

Comment:
@vickywane the path depends on how the code is installed. Instead of changing the default config, the correct approach is to use the correct setup scripts for different environments. We discussed on the roundtable call. See how we build rpi OS image and similarly create a separate setup script for dev simulation of edge install.

Reply:
I would take note of this.